source/extensions/filters/http/lua/lua_filter.cc

```diff
@@ -485,6 +430,81 @@ int StreamHandleWrapper::luaImportPublicKey(lua_State* state) {
   return 1;
 }
 
+FireAndForgetHttpWriter::FireAndForgetHttpWriter(Filter& filter) : filter_(filter) {}
+
+int FireAndForgetHttpWriter::luaHttpCallNonblocking(lua_State* state) {
+  LuaFilterUtil::makeHttpCall(state, filter_, *this);
+  return 0;
+}
+
+Http::AsyncClient::Request*
+LuaFilterUtil::makeHttpCall(lua_State* state, Filter& filter,
+                            Http::AsyncClient::Callbacks& callbacksListener) {
```

Comment:
nit: callbacksListener -> callbacks (or at the very least snake case parameter names)